<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<<<<<<< Updated upstream
<testsuite hostname="sumi" name="Default test" tests="1" failures="0" timestamp="7 May 2017 17:13:19 GMT" time="80.606" errors="0">
  <testcase name="run" time="27.955" classname="automationFramework.POM_TC"/>
=======
<testsuite hostname="shapan-HP" name="Default test" tests="1" failures="0" timestamp="7 May 2017 15:34:04 GMT" time="0.48" errors="0">
  <testcase name="@BeforeTest beforeTest" time="0.0" classname="automationFramework.Module_TC">
    <failure type="java.io.FileNotFoundException" message="C:\QA_Selenium\charlotte\OnlineStore.com\src\test\java\TestData.xlsx (The system cannot find the file specified)">
      <![CDATA[java.io.FileNotFoundException: C:\QA_Selenium\charlotte\OnlineStore.com\src\test\java\TestData.xlsx (The system cannot find the file specified)
	at java.io.FileInputStream.open0(Native Method)
	at java.io.FileInputStream.open(Unknown Source)
	at java.io.FileInputStream.<init>(Unknown Source)
	at java.io.FileInputStream.<init>(Unknown Source)
	at utility.ExcelUtils.SetExcelFile(ExcelUtils.java:23)
	at automationFramework.Module_TC.beforeTest(Module_TC.java:28)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:564)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:213)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:138)
	at org.testng.TestRunner.beforeRun(TestRunner.java:641)
	at org.testng.TestRunner.run(TestRunner.java:609)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
    </failure>
  </testcase> <!-- @BeforeTest beforeTest -->
  <testcase name="@AfterTest afterTest" time="0.0" classname="automationFramework.Module_TC">
    <skipped/>
  </testcase> <!-- @AfterTest afterTest -->
  <testcase name="run" time="0.004" classname="automationFramework.Module_TC">
    <skipped/>
  </testcase> <!-- run -->
>>>>>>> Stashed changes
</testsuite> <!-- Default test -->
